src/chaosinventory/fallback.cfg

```diff
@@ -0,0 +1,3 @@
+[django]
```

Comment:
As said in the other comment, this file should not be added as it's out of scope